newIDE/app/src/AssetStore/AssetPackTiles.js

```diff
@@ -0,0 +1,366 @@
+|}) => {
+  const classesForGridListItem = useStylesForGridListItem();
+  return (
+    <GridListTile
+      classes={classesForGridListItem}
+      tabIndex={0}
+      onKeyPress={(event: SyntheticKeyboardEvent<HTMLLIElement>): void => {
+        if (shouldValidate(event)) {
+          onSelect();
+        }
+      }}
+      style={style}
+      onClick={onSelect}
+    >
+      <Paper
+        id={`asset-pack-${assetPack.tag.replace(/\s/g, '-')}`}
+        elevation={2}
+        style={styles.paper}
+        background="light"
+      >
+        <CorsAwareImage
+          key={assetPack.name}
+          style={styles.previewImage}
+          src={assetPack.thumbnailUrl}
+          alt={`Preview image of asset pack ${assetPack.name}`}
+        />
+        <Column>
+          <Line justifyContent="space-between" noMargin>
+            <Text style={styles.packTitle} size="body2">
+              {assetPack.name}
+            </Text>
+            <Text style={styles.packTitle} color="primary" size="body2">
+              <Trans>{assetPack.assetsCount} Assets</Trans>
+              {assetPack.userFriendlyPrice
+                ? ' - ' + assetPack.userFriendlyPrice
+                : null}
+            </Text>
+          </Line>
+        </Column>
+      </Paper>
+    </GridListTile>
+  );
+};
+
+export const PrivateAssetPackTile = ({
+  assetPackListingData,
+  onSelect,
+  style,
+  owned,
+}: {|
+  assetPackListingData: PrivateAssetPackListingData,
+  onSelect: () => void,
+  /** Props needed so that GridList component can adjust tile size */
+  style?: any,
+  owned: boolean,
+|}) => {
+  const classesForGridListItem = useStylesForGridListItem();
+  return (
+    <GridListTile
+      classes={classesForGridListItem}
+      tabIndex={0}
+      onKeyPress={(event: SyntheticKeyboardEvent<HTMLLIElement>): void => {
+        if (shouldValidate(event)) {
+          onSelect();
+        }
+      }}
+      style={style}
+      onClick={onSelect}
+    >
+      <Paper elevation={2} style={styles.paper} background="light">
+        <CorsAwareImage
+          key={assetPackListingData.name}
+          style={styles.previewImage}
+          src={assetPackListingData.thumbnailUrls[0]}
+          alt={`Preview image of asset pack ${assetPackListingData.name}`}
+        />
+        <div style={styles.priceTagContainer}>
+          <PrivateAssetPackPriceTag
+            privateAssetPackListingData={assetPackListingData}
+            withOverlay
+            owned={owned}
+          />
+        </div>
+        <Column>
+          <Line justifyContent="space-between" noMargin>
+            <Text style={styles.packTitle} size="body2">
+              {assetPackListingData.name}
+            </Text>
+            <Text style={styles.packTitle} color="primary" size="body2">
+              {assetPackListingData.description}
+            </Text>
+          </Line>
+        </Column>
+      </Paper>
+    </GridListTile>
+  );
+};
+
+export const PromoBundleAssetPackTile = ({
```

Comment:
It's not wrapped in a GridListTile like the other components, should it be named tile?

Reply:
You're right, I wanted to use a tile, but ended up no doing it. I'll rename this